mmda/predictors/lp_predictors.py

```diff
@@ -18,7 +19,7 @@ def __init__(self, model: lp.Detectron2LayoutModel):
     @classmethod
     def from_pretrained(
         cls,
-        config_path: str,
+        config_path: str = 'lp://PrimaLayout/mask_rcnn_R_50_FPN_3x/config',
```

Comment:
@lolipopshock What set of pre-trained weights do you recommend we use for production SPP?

Reply:
Thanks! I am updating layoutparser to include the some newer options here that are optimized for production. Will share with you later today.